mlflow/utils/validation.py

```diff
@@ -639,3 +640,16 @@ def _validate_experiment_artifact_location_length(artifact_location: str):
             "MLFLOW_ARTIFACT_LOCATION_MAX_LENGTH environment variable.",
             INVALID_PARAMETER_VALUE,
         )
+
+
+def _validate_logged_model_name(name: Optional[str]) -> None:
+    if name is None:
+        return
+
+    forbidden_chars = ("/", ":", ".")
```

Comment:
is "\" allowed?

Reply:
No

Reply:
let me check if that's allowed on databricks.